images/ceph/Dockerfile

```diff
@@ -12,12 +12,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM BASEIMAGE
+FROM centos:7
```

Comment:
We could also have an entrypoint for rook in ceph-container so you don't have to use tini

Reply:
If you use ceph/ceph, we are sharing much more code, bugs, versions with all other ceph users increasing the reliability & support for the whole community.

Reply:
@ErwanAliasr1 agreed, the first option is sounding the best for those reasons. 

Regarding tini, rook does need to launch other processes/tools while it is orchestrating the cluster. Without tini, we could have orphaned processes without a proper PID 1 implementation. But when Rook runs the ceph/ceph daemon pods, it will not use `tini` in that scenario.

Reply:
ceph-container would be able to make it in way we manage the pid 1 while adding some supplementary debugging/support tricks like recovering a shell when a containers dies.

Reply:
@ErwanAliasr1 What do you mean with "recovering a shell when a container dies"? If Ceph dies inside the container the container should exit so that Kubernetes can do "it's thing".

Reply:
@ErwanAliasr1 That is not good for containers though. If Ceph exits, the container should also exit and be restarted by Kubernetes.
If the container would "hang" endlessly (need manual intervention), that is a problem.